src/features/ui/components/Button/index.ts

```diff
@@ -36,7 +36,8 @@ const sizes = {
 
   small: css`
     ${typography.label.medium}
-    padding: 0.3em 2em 0.2em;
+    padding: 0.3em 0 0.2em;
+    min-width: 10rem;
```

Comment:
Interesting enough, the horizontal padding was not used anywhere in the application besides the event action button.

This change ensures we stick to the layout's `10rem`, but keep the `small` variant resilient enough.